cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -132,6 +132,71 @@ template <typename T>
 class DirichletBC
 {
 private:
+  /// @brief Build list of cells (local to process, including ghosts)
+  /// which contain any of the input degrees of freedom.
```

Comment:
This is somewhat ambiguous - elsewhere we only associate interior dofs with the 'cell'. I presume you mean cell closure?